searchcore/src/vespa/searchcore/proton/feedoperation/updateoperation.cpp

```diff
@@ -73,7 +72,7 @@ UpdateOperation::deserialize(vespalib::nbostream &is, const DocumentTypeRepo &re
 {
     DocumentOperation::deserialize(is, repo);
     try {
-        deserializeUpdate(is, repo);
+        deserializeUpdate(std::move(is), repo);
```

Comment:
Should `UpdateOperation::deserialize` be altered to take in an rvalue as well?

Reply:
At least not now. It is implementing and interface.